wicketd/src/update_events.rs

```diff
@@ -89,20 +90,21 @@ pub struct UpdateEvent {
 #[derive(Clone, Debug, JsonSchema, Serialize)]
 #[serde(rename_all = "snake_case", tag = "kind", content = "data")]
 pub enum UpdateEventKind {
-    Success(UpdateEventSuccessKind),
-    Failure(UpdateEventFailureKind),
+    NormalEvent(UpdateNormalEventKind),
```

Comment:
Nit: I'd probably shorten these to variants to just `Normal` and `Terminal` | `Failure`, since the types inside provide a good amount of info.